Dockerfile

```diff
@@ -1,21 +1,21 @@
-FROM node:18
+FROM node:22

```

Comment:
We should update in all places for consistency if this is meant to be the new base version.